modules/nf-core/cnvkit/genemetrics/main.nf

```diff
@@ -0,0 +1,40 @@
+process CNVKIT_GENEMETRICS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::cnvkit=0.9.9 bioconda::samtools=1.16.1"
```

Comment:
There should be a new container available with 0.9.10 and samtools 1.17 that we use for batch now.